buildkite/build_project.sh

```diff
@@ -211,7 +211,9 @@ case "$REPO_FULL_NAME" in
         dub test "--compiler=$DC"
         ;;
 
-    sociomantic-tsunami/ocean|sociomantic-tsunami/turtle)
+    sociomantic-tsunami/ocean | \
+    sociomantic-tsunami/turtle | \
+    sociomantic-tsunami/swarm)
```

Comment:
This could have been done more future proof by using a case like `sociomantic-tsunami/*)`.

Reply:
I might do that in the next PR, when adding the dhtproto.